content/webapp/components/HTMLSerializers/HTMLSerializers.tsx

```diff
@@ -13,7 +13,15 @@ export const dropCapSerializer: HTMLSerializer<ReactElement> = (
   children,
   key
 ) => {
-  if (type === Elements.paragraph && key === '0' && children[0] !== undefined) {
+  // Note: `key.toString() === '0'` might look strange, because isn't `key`
+  // already a string?
+  //
+  // For some reason, we're getting passed `key` as a number here, so we
+  // need this or drop caps don't work correctly.
+  //
+  // TODO: Work out why this isn't being caught by the type checker elsewhere,
+  // and fix it properly.
+  if (type === Elements.paragraph && (key === '0' || key.toString() === '0') && children[0] !== undefined) {
```

Comment:
I'd be surprised if this didn't need a bit of a lint :trollface:

Reply:
I guess the `key === '0'` is surplus to requirements here also

Reply:
Good shout!

Reply:
How about now?